[elogind] Failed to build

```
[39/369] Compiling C object src/basic/libbasic.a.p/btrfs-util.c.o
FAILED: src/basic/libbasic.a.p/btrfs-util.c.o 
cc -Isrc/basic/libbasic.a.p -Isrc/basic -I../src/basic -I../src/boot -Isrc/shared -I../src/shared -Isrc/systemd -I../src/systemd -Isrc/journal -I../src/journal -Isrc/login -I../src/login -Isrc/core -I../src/core -I../src/libelogind/sd-bus -I../src/libelogind/sd-device -I../src/libelogind/sd-event -I../src/libelogind/sd-hwdb -I../src/libelogind/sd-id128 -Isrc/sleep -I../src/sleep -Isrc/update-utmp -I../src/update-utmp -I. -I.. -flto=auto -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=gnu99 -O3 -Wno-unused-parameter -Wno-missing-field-initializers -Wno-unused-result -Wno-format-signedness -Werror=undef -Wlogical-op -Wmissing-include-dirs -Wold-style-definition -Wpointer-arith -Winit-self -Wfloat-equal -Wsuggest-attribute=noreturn -Werror=missing-prototypes -Werror=implicit-function-declaration -Werror=missing-declarations -Werror=return-type -Werror=incompatible-pointer-types -Werror=format=2 -Wstrict-prototypes -Wredundant-decls -Wmissing-noreturn -Wimplicit-fallthrough=5 -Wshadow -Wendif-labels -Wstrict-aliasing=2 -Wwrite-strings -Werror=overflow -Werror=shift-count-overflow -Werror=shift-overflow=2 -Wdate-time -Wnested-externs -Wno-maybe-uninitialized -ffast-math -fno-common -fdiagnostics-show-option -fno-strict-aliasing -fvisibility=hidden -fstack-protector -fstack-protector-strong --param=ssp-buffer-size=4 -ffunction-sections -fdata-sections -Werror=shadow -include config.h -fuse-ld=gold -fPIC -pthread -fvisibility=default -MD -MQ src/basic/libbasic.a.p/btrfs-util.c.o -MF src/basic/libbasic.a.p/btrfs-util.c.o.d -o src/basic/libbasic.a.p/btrfs-util.c.o -c ../src/basic/btrfs-util.c
../src/basic/btrfs-util.c:6:10: fatal error: linux/btrfs_tree.h: No such file or directory
    6 | #include <linux/btrfs_tree.h>
      |          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
[40/369] Compiling C object src/basic/libbasic.a.p/blockdev-util.c.o
[41/369] Compiling C object src/libelogind/libelogind_static.a.p/sd-bus_bus-objects.c.o
[42/369] Compiling C object src/basic/libbasic.a.p/cgroup-util.c.o
[43/369] Compiling C object src/basic/libbasic.a.p/bus-label.c.o
[44/369] Compiling C object src/libelogind/libelogind_static.a.p/sd-bus_bus-message.c.o
ninja: build stopped: subcommand failed.
elogind failed to build: `ninja -C builddir` exited with 1
```

--- a/packages/elogind.rb
+++ b/packages/elogind.rb
@@ -3,22 +3,23 @@ require 'package'
 class Elogind < Package
   description 'Standalone systemd-logind fork'
   homepage 'https://github.com/elogind/elogind'
-  version '246.9.2-4c20'
+  @_ver = '246.10'
+  version @_ver
   compatibility 'all'
-  source_url 'https://github.com/elogind/elogind/archive/4c2042edeb73823c5e7c831c34bea942b9525027.zip'
+  source_url "https://github.com/elogind/elogind/archive/v#{@_ver}.tar.gz"
   source_sha256 '9a6285c077f3345c69fc21d1902226966ef6dc6e40d0eba259ee882c5b766eca'
 
   binary_url({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/elogind-246.9.2-4c20-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/elogind-246.9.2-4c20-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/elogind-246.9.2-4c20-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/elogind-246.9.2-4c20-chromeos-x86_64.tar.xz'
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/elogind-246.10-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/elogind-246.10-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/elogind-246.10-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/elogind-246.10-chromeos-x86_64.tar.xz'
   })
   binary_sha256({
-    aarch64: '4978fe7be484892e87a570c6fb8bc4d37c0f75f498178b298f320af25667fb0a',
-     armv7l: '4978fe7be484892e87a570c6fb8bc4d37c0f75f498178b298f320af25667fb0a',
-       i686: '0b95082c824a6234a2bdd292c741a8c614f1c58afc16375bab9e7091f8f63126',
-     x86_64: '9ba1128c658fcb39a903986ad10ed37121f6e0281b081987a8e6fe0a65dc14f4'
+    aarch64: '757fc7dc87b8699b51abdbc3b5b7e7dc505cc3bf1934aabe356d7bfff6712d2d',
+     armv7l: '757fc7dc87b8699b51abdbc3b5b7e7dc505cc3bf1934aabe356d7bfff6712d2d',
+       i686: '8e1c7f42275654fe1a3010c8631be0b5095f6bdcad3020f5ed4b3dd68289eee7',
+     x86_64: '186e7ec40bded8ae9624b8dd3ff90880cd6d0de98016413745d2f672a43cbfa7'
   })
 
   depends_on 'eudev'
@@ -31,6 +32,7 @@ class Elogind < Package
   depends_on 'libxslt'
   depends_on 'shadow'
   depends_on 'glib'
+  depends_on 'polkit'
 
   def self.build
     system "meson #{CREW_MESON_LTO_OPTIONS} \

--- a/packages/elogind.rb
+++ b/packages/elogind.rb
@@ -3,11 +3,24 @@ require 'package'
 class Elogind < Package
   description 'Standalone systemd-logind fork'
   homepage 'https://github.com/elogind/elogind'
-  version '246.9.2'
+  version '246.9.2-4c20'
   compatibility 'all'
-  source_url 'https://github.com/elogind/elogind/archive/v246.9.2.tar.gz'
-  source_sha256 'dd2fcf22a89a078cad22e633d2f14a4cc9f4a9c8bae25c0e39fc4aec3e273bc9'
-  
+  source_url 'https://github.com/elogind/elogind/archive/4c2042edeb73823c5e7c831c34bea942b9525027.zip'
+  source_sha256 '9a6285c077f3345c69fc21d1902226966ef6dc6e40d0eba259ee882c5b766eca'
+
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/elogind-246.9.2-4c20-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/elogind-246.9.2-4c20-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/elogind-246.9.2-4c20-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/elogind-246.9.2-4c20-chromeos-x86_64.tar.xz'
+  })
+  binary_sha256({
+    aarch64: '4978fe7be484892e87a570c6fb8bc4d37c0f75f498178b298f320af25667fb0a',
+     armv7l: '4978fe7be484892e87a570c6fb8bc4d37c0f75f498178b298f320af25667fb0a',
+       i686: '0b95082c824a6234a2bdd292c741a8c614f1c58afc16375bab9e7091f8f63126',
+     x86_64: '9ba1128c658fcb39a903986ad10ed37121f6e0281b081987a8e6fe0a65dc14f4'
+  })
+
   depends_on 'eudev'
   depends_on 'libcap'
   depends_on 'libseccomp'
@@ -22,9 +35,10 @@ class Elogind < Package
   def self.build
     system "meson #{CREW_MESON_LTO_OPTIONS} \
             -Drootprefix=#{CREW_PREFIX} \
+            -Dsysconfdir=#{CREW_PREFIX}/etc \
             --strip \
             -Dcgroup-controller=elogind \
-            -Dman=false \
+            -Dman=true \
             -Ddefault-hierarchy=legacy \
             -Ddefault-kill-user-processes=false \
             -Dhalt-path=/sbin/halt \
@@ -33,7 +47,7 @@ class Elogind < Package
             -Drootbindir=#{CREW_PREFIX}/bin \
             -Dreboot-path=/sbin/reboot \
             builddir"
-    system "ninja -C builddir"
+    system 'ninja -C builddir'
   end
 
   def self.install

